bedrock/firefox/templates/firefox/whatsnew/whatsnew-50.html

```diff
@@ -0,0 +1,46 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/mobile-download-desktop.html" %}
+
+{% set send_to_device_message_set = 'fx-50-whatsnew' %}
+{% set app_store_url = 'https://app.adjust.com/2uo1qc?campaign=whats_new&amp;adgroup=iOS_App_Store&amp;creative=badge_en&amp;fallback=https%3A%2F%2Fitunes.apple.com%2Fapp%2Fapple-store%2Fid989804926%3Fpt%3D373246%26ct%3Dadjust_tracker%26mt%3D8' %}
+{% set play_store_url = 'https://app.adjust.com/2uo1qc?campaign=whats_new&amp;adgroup=Google_Play_Store&amp;creative=badge_en' %}
+
+{% add_lang_files "firefox/sendto" "download" %}
+
+{% block gtm_page_id %}data-gtm-page-id="/firefox/whatsnew/"{% endblock %}
+
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{% block optimizely %}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_whatsnew_50' %}
+{% endblock %}
+
+{% block headline %}
+
+{# If user is in a locale with translated basket messages... #}
+{% if show_send_to_device %}
+  {# Custom headlines for en-* users #}
+  {% if LANG.startswith('en-') %}
+    <h4 class="up-to-date"><span>Your Firefox is up to date.</span></h4>
+    <h2>Now get on the go.</h2>
+
+    <h1>Send Firefox to your phone<br> and unleash your Internet.</h1>
```

Comment:
Not a big deal as this is pretty much a desktop only page, but you could set the `<br>` here to `display:none;` at smaller view ports to avoid the odd looking line break in the copy. Feel free to ignore if you think this isn't important.